Tests/TestUtilities/TempFile.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.IO;
+
+namespace TestUtilities
+{
+    public sealed class TempFile : IDisposable
+    {
+        public string FileName { get; }
+
+        public TempFile()
+        {
+            FileName = Path.GetRandomFileName();
+        }
```

Comment:
these can also be joined together
```suggestion
		public string FileName { get; } = Path.GetRandomFileName();
```